dataproc_jupyter_plugin/tests/test_dataproc.py

```diff
@@ -13,29 +13,54 @@
 # limitations under the License.
 
 import json
+from unittest import mock
 
 from dataproc_jupyter_plugin.tests import mocks
 
+from google.cloud.dataproc_v1.services.cluster_controller import (
+    ClusterControllerClient,
+    pagers
+)
+from google.auth import credentials as ga_credentials
+from google.cloud.dataproc_v1.types import clusters
+import pytest
 
-async def test_list_clusters(monkeypatch, jp_fetch):
-    mocks.patch_mocks(monkeypatch)
 
-    mock_project_id = "credentials-project"
-    mock_page_token = "mock-page-token"
-    mock_region_id = "mock-region"
-    mock_page_size = "mock_page_size"
-    response = await jp_fetch(
-        "dataproc-plugin",
-        "clusterList",
-        params={"pageSize": mock_page_size, "pageToken": mock_page_token},
-    )
```

Comment:
This call needs to still be in the updated test. This is the entire point of the test.